redash/models/parameterized_query.py

```diff
@@ -142,5 +153,6 @@ def text(self):
 
 class InvalidParameterError(Exception):
     def __init__(self, parameters):
-        message = u"The following parameter values are incompatible with their definitions: {}".format(", ".join(parameters))
+        parameter_names = ", ".join(parameters)
```

Comment:
```suggestion
        parameter_names = u", ".join(parameters)
```

Reply:
It doesn't look like you applied the suggestion?

Reply:
Ping.

Reply:
Somehow missed that one!